feat(features): Remove the need to update getsentry

wmak opened the pull request:
- We currently have to update getsentry whenever we want to use flagr
  our entity handler
- this adds a third parameter to `add` that determines whether the
  feature should be handled by the entity handler
- `organizations:slack-allow-workspace` is no longer used

wmak committed:
feat(features): Remove the need to update getsentry

- We currently have to update getsentry whenever we want to use flagr
  our entity handler
- this adds a third parameter to `add` that determines whether the
  feature should be handled by the entity handler

k-fish reviewed (approved):
Big ty for this one, it was always extra steps to remember 👏

wmak committed:
Merge branch 'master' into wmak/feat/simplify-entity-handlers

Zylphrex reviewed (approved):
👏 this is super helpful, thanks!

k-fish committed:
ref(vis): Add upcoming feature flags (#25879)

Adding flags I'm guessing we'll need ahead of time, figured I'd add the rest of the ones I know about too.
